eli5/sklearn/explain_prediction.py

```diff
@@ -23,8 +23,8 @@
 
 
 @singledispatch
-def explain_prediction(clf, vec, doc, top=_TOP, class_names=None,
-                       feature_names=None, vectorized=False):
+def explain_prediction(clf, doc, vec=None, top=_TOP, class_names=None,
+                       feature_names=None, target_names=None):
```

Comment:
probably it makes sense to rename class_names to target_names and keep a single argument for them; target_names is the name used in sklearn.